templates/.github/workflows/stage-pipeline.yml

```diff
@@ -0,0 +1,76 @@
+# .github/workflows/stage-pipeline.yml
+# This is a github action pipeline for staging. 
+name: Stage Deployment
+on:
+  push:
+    branches: [ master ]
+# Jobs
+jobs:
+  build-stage:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+      build_env: staging
+    steps:
+    # Checkout the code
+      - uses: actions/checkout@v2
+    # Run Tests
+      - name: Run Unit Tests
+        uses: actions/setup-node@v1
+        with: 
+          node-version: 12.x
+      - run: |
+          yarn
+          yarn test --watchAll=false
+    # Build Code
+      - name: Build Static Site
+        uses: actions/setup-node@v1
+        with: 
+          node-version: 12.x  
+      - run: |
+          yarn
+          yarn build
+          ls -al
+        env:
+          REACT_APP_CONFIG: $ {{ env.build_env }}
+    # Upload build to github artifact
+      - name: Upload build artifact to Github
+        uses: actions/upload-artifact@v1
+        with:
+          name: stage-build-artifact
+          path: build/
+# TODO: Deploy for stage
+  deploy-stage:
+    name: Upload to Amazon S3
+    runs-on: ubuntu-latest
+    needs: build-stage
+    env:
+      region: us-west-2
+      bucket: <% bucket %> 
```

Comment:
This is not a valid template var

Reply:
right caught that forgot to push my latest changes.